server/jersey-server/src/test/java/com/quorum/tessera/server/jaxrs/JerseyServerIT.java

```diff
@@ -39,12 +53,22 @@ public void onTearDown() throws Exception {
 
     @Test
     public void ping() {
+        
+        MultivaluedMap<String, Object> headers = new MultivaluedHashMap<>();
+        headers.add("Origin", "*.acme.com");
```

Comment:
wow - I was not expecting wildcards in the Origin header... I would always have expected that the Origin is exactly the site name from where the script attempting the CORS request has been downloaded.